apps/docs/content/docs/components/modal.mdx

```diff
@@ -137,6 +137,18 @@ Modal offers a `motionProps` property to customize the `enter` / `exit` animatio
 
 > Learn more about Framer motion variants [here](https://www.framer.com/motion/animation/#variants).
 
+### Draggable
+
+Try to drag the header part.
+
+<CodeDemo title="Draggable" files={modalContent.draggable} />
+
+### Draggable Overflow
+
+Set draggable to true to drag, the default is true. Set overflow to true can drag overflow the viewport.
```

Comment:
> Set draggable to true to drag, the default is true.

I don't think we need to mention this because by default it's already `true` and I don't think there is a case to use `useDraggable` but disabling dragging. Or any use case in your mind that we need `draggable = false` while using `useDraggable`?

Reply:
Agree, I have change this description to `Set overflow to true can drag overflow the viewport.`.

But I think the draggable parameter can be retained, which gives the user the ability to control whether the model is dragged.